apps/api/src/app/health/health.controller.ts

```diff
@@ -16,9 +16,6 @@ import { HealthService } from './health.service';
 export class HealthController {
   public constructor(private readonly healthService: HealthService) {}
 
-  @Get()
```

Comment:
⚠️ This will destroy the [health check endpoint](https://github.com/ghostfolio/ghostfolio#health-check-experimental)!

What about this?

```
@Get()
@HttpCode(HttpStatus.OK)
public async getHealth() {
  return { status: getReasonPhrase(HttpStatus.OK) };
}
```

Reply:
I see. I was wondering where it is actually used for. I send a request to it and no response came back so i thought it is not used and only the other health check endpoints in that file are used anymore as they actually return a response.
In that case we should actually make it really return a 200 status code like described in the link you provided.

Reply:
I have extended the documentation by `status: "OK"`